pages/blog/v08-release.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Grouparoo v0.8 release"
+date: "2022-01-31"
+author: "Brian Leonard"
+pullQuote: "Grouparoo's v0.8 beings a new Model UI and other improvements."
+image: "v08-release/video.jpg"
+tags: [product, engineering]
+---
+
+The v0.8 release is our first major iteration on the UI for creating your data pipeline. In the v0.7 release, we added Models, which allowed data engineers to sync multiple data schemas to Destinations. This release summarizes those Models better in the UI, giving you a clearer overview of the configuration.
+
+Here are the key features of the release.
+
+- Model-centric UI to see on one page where the data comes from, what it looks like, and where it is going
```

Comment:
Not major: I wonder if "Model-centric UI" is too much of an internal term or phrasing.
"Model-based UI"
"Model-focused UI"

Wondering if maybe I'm too close to it though.